mpa/modules/hooks/__init__.py

```diff
@@ -20,3 +20,4 @@
 from . import logger_replace_hook
 from . import auxiliary_hooks
 from . import save_initial_weight_hook
+from . import fp16_sam_optimizer_hook
```

Comment:
No new line for this code; Should we register hook on here, instead of submodule?

Reply:
I can't understand, what do you mean by submodule?

Reply:
Sorry, I confused. Please add new line for this.